### PR TITLE
【prim】New layer_norm grad

### PR DESCRIPTION
**Pull request by xiaoguoguo626807**
### PR types
Others 

### PR changes
Others 

### Describe

Pcard-66975
layer_norm vjp 规则

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py [hunk 1]
```diff
@@ -237,10 +237,12 @@ def test_train(self):
 
     def test_train_composite(self):
         core._set_prim_backward_enabled(True)
+        # core._add_skip_comp_ops("layer_norm")
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py:240]: Remove comments

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py [hunk 2]
```diff
@@ -237,10 +237,12 @@ def test_train(self):
...
         static_loss, static_ppl = self.train_static(
             self.bert_config, self.data_reader
         )
         core._set_prim_backward_enabled(False)
+        # core._add_skip_comp_ops("layer_norm")
```
Comment (JiabinYang) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py:245]: same

**Review by JiabinYang — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py (hunk 1 above)
Comment (xiaoguoguo626807) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py:240]: modify in next PR

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py (hunk 2 above)
Comment (xiaoguoguo626807) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/test_bert.py:245]: same

**Pull request merged**